### PR TITLE
Added support for Incognito mode

### PR DESCRIPTION
**Pull request by srm09**

### CODE REVIEW

File: js/shell.js [hunk 1]
```diff
@@ -89,6 +89,9 @@ window.onload = function() {
         this.label = label;
         this.iframe = iframe;
 
+        this.makePrivate = function() {
```
Comment (jdm) [inline js/shell.js:92]: I don't think we need a separate function for this.

File: index.html [hunk 2]
```diff
@@ -15,6 +15,7 @@
                 <div id="header-placeholder"></div>
                 <ul id="tab-container">
                     <li id="new-tab" class="tab tab-new" onclick="onNewTab()"><i class="fa fa-plus tab-new-icon"></i></li>
+                    <li id="new-tab" class="tab tab-new" onclick="onNewPrivateTab()" title="Private/Incognito mode"><i class="fa fa-plus tab-new-private-icon"></i></li>
```
Comment (jdm) [inline index.html]: Let's have onNewTab take a boolean argument and make both call onNewTab instead.

**Comment by jdm** [comment]
Looks pretty good! Just a few small changes to reduce the code duplication.

File: js/shell.js (hunk 1 above)
Comment (srm09) [inline js/shell.js:92]: Wrote this function to prevent direct access to the `iframe` attribute.

Comment (jdm) [inline js/shell.js:92]: Oh, my mistake. Carry on!

File: js/shell.js [hunk 3]
```diff
@@ -169,6 +172,9 @@ window.onload = function() {
         var tabContainer = document.getElementById("tab-container");
         tabContainer.insertBefore(tab.label, newTabButton);
 
+        if(is_private)
```
Comment (jdm) [inline js/shell.js]: nit: space before (

**Comment by jdm** [comment]
@glennw Could you merge this?

**Pull request merged**